frontend/public/style/_vars.scss

```diff
@@ -19,26 +19,13 @@ $fa-font-path: $consoleWebDistPath + '/';
 // Use PatternFly 4 font family in PatternFly 3.
 $font-family-base: var(--pf-global--FontFamily--sans-serif);
```

Comment:
This can be removed and any existing instances just changed to `var(--pf-global--FontFamily--sans-serif)`.

Reply:
removed